x-pack/plugins/endpoint/public/applications/endpoint/components/page_view.tsx

```diff
@@ -20,24 +20,65 @@ import React, { memo, ReactNode } from 'react';
 import styled from 'styled-components';
 
 const StyledEuiPage = styled(EuiPage)`
-  padding: 0;
+  &.endpoint--isListView {
+    padding: 0;
 
-  .endpoint-header {
-    padding: ${props => props.theme.eui.euiSizeL};
+    .endpoint-header {
+      padding: ${props => props.theme.eui.euiSizeL};
+    }
+    .endpoint-page-content {
+      border-left: none;
+      border-right: none;
+    }
   }
-  .endpoint-page-content {
-    border-left: none;
-    border-right: none;
+  &.endpoint--isDetailsView {
```

Comment:
i see this `--` pattern a lot in other css stuffs, does it have a standard meaning? -- pliss teach me

Reply:
:)
Ohhh. I hope I get this right from memory.
In CSS there are a few patterns for managing styles, one is called BEM - Block-Element-Modifier. in that pattern, a Modifier classname is normally defined using the double dash `--` (in a past project, the `_` was used, but same concept) . Its purpose is to normally drive how other classnames below it will adjust based on its presence. 

(there are a few other patterns out there - google `BEM vs` and you should see the others popup)
😃